Clicker/Models/ListDiffableModels/Poll.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  Poll.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 2/3/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+enum PollState {
+    case live
+    case ended
+    case shared
+}
+
+class Poll {
+    
+    var id: Int?
+    var text: String
+    var questionType: QuestionType
+    var options: [String]
+    var results: [String:Any]
+    // results format:
```

Comment:
move state to be above results, so the results comment is at the end of the block